src/Extension/CoreExtension.php

```diff
@@ -915,6 +916,20 @@ public static function keys($array): array
         return array_keys($array);
     }
 
+    /**
+     * Invokes a callable
+     *
+     * @param callable $callable
+     * @param ...$arguments
+     * @return mixed
+     *
+     * @internal
+     */
+    public static function invoke(callable $callable, ...$arguments): mixed
+    {
+        return $callable(...$arguments);
```

Comment:
Don't we want to limit / check the `$callable` argument ? 

Via `checkArrow()` for instance?

Reply:
The code now looks more like the other functions/filters that accept arrow functions, including a call to `checkArrow()`